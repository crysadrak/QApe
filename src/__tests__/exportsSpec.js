import qapeExports from '../exports';

describe('Exports', () => {
    it('are all available', () => {
        expect(qapeExports).toMatchSnapshot();
    });
});
