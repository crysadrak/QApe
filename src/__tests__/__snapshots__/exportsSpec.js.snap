// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exports are all available 1`] = `
Object {
  "AbstractAction": [Function],
}
`;
